coordination.md

```diff
@@ -0,0 +1,72 @@
+# Coordination
```

Comment:
```suggestion
# Coordination 🚧
```

My impression is that this is again writing toward a direction that many of us hope we move in, but not things that have emerged as validated practice yet. Would love to indicate that again via this emoji :)